src/api/impl/ConnectedServer.cpp

```diff
@@ -26,19 +26,19 @@ _NODISCARD static bool HandleHttpError(_In_ const ra::services::Http::Response&
     return false;
 }
 
-_NODISCARD static bool GetJson(_In_ const char* sApiName, _In_ const ra::services::Http::Response& httpResponse, _Inout_ ApiResponseBase& pResponse, _Out_ rapidjson::Document& pDocument) noexcept
+_NODISCARD static bool GetJson([[maybe_unused]] _In_ const char* sApiName, _In_ const ra::services::Http::Response& httpResponse, _Inout_ ApiResponseBase& pResponse, _Out_ rapidjson::Document& pDocument) noexcept
 {
     if (HandleHttpError(httpResponse, pResponse))
     {
-        pDocument.Clear();
+        pDocument.SetArray(); // Two Debug Assertion Errors, one from RapidJSON and one from Kernel
 
         RA_LOG_ERR("-- %s: %s", sApiName, pResponse.ErrorMessage.c_str());
         return false;
     }
 
     if (httpResponse.Content().empty())
     {
-        pDocument.Clear();
+        pDocument.SetArray(); // Debug Assertion Error
```

Comment:
These two where causing debug assertion failures. According the documentation, `rapidjson::Document::SetArray`, sets the Document as an empty array.

Reply:
I agree with the changes, but please remove the comments. They're review explanations, not describing the committed behavior.